cmd/nomos/status/client.go

```diff
@@ -168,6 +168,9 @@ func (c *ClusterClient) clusterStatus(ctx context.Context, cluster, namespace st
 
 	if namespace != "" {
 		c.namespaceRepoClusterStatus(ctx, cs, namespace)
+		if namespace == "config-management-system" {
+			c.multiRepoClusterStatus(ctx, cs)
```

Comment:
`c.multiRepoClusterStatus` lists all RootSyncs and RepoSyncs, but you only need RootSyncs here.

Reply:
as stated in previous comment, this has been split and updated to show only root